Makefile

```diff
@@ -1016,6 +1016,9 @@ install: version $(TARGET)
 	cp -R --no-preserve=ownership data/help $(DATA_PREFIX)
 ifeq ($(TILES), 1)
 	cp -R --no-preserve=ownership gfx $(DATA_PREFIX)
+	install -Dm755 -t /app/share/applications/ data/xdg/org.cataclysmdda.CataclysmDDA.desktop
```

Comment:
What if you are not using Flatpak?

Reply:
These files are not flatpak-specific.  You would want them in any freedesktop system.

Reply:
Oh, I understand now.  Sorry -- I had just copied the files from the flatpak manifest and did not make the installation path variable.  I will push a fix